src/spectrum.rs

```diff
@@ -0,0 +1,143 @@
+/// ! Conversions from spectrum intensities to XYZ
+///
+
+use xyz::Xyz;
+use num::{Float,Zero};
+
+
+/// Data from http://www.cvrl.org/cmfs.htm, CIE 1931 2-deg, XYZ CMFs
+/// 360 nm -> 830 nm += 5nm
+const SPECTRUM_TO_XYZ_MAP: [[f32; 3]; 95] = [
```

Comment:
I'm fairly sure it makes no difference, the compiler already knows the size of the array.